scripts/deployToDB.py

```diff
@@ -165,8 +123,53 @@ def update_config(data_diff, selector):
     return json.dumps(results, indent=2)
 
 
+def validate_acc_def_diff(diff):
```

Comment:
we are mixing up things here - validation of whether destination definition has right reference of account definition should be part of test suite which runs as part of PR
deploy python script responsibility is to check for changes in account definitions & accordingly create/update them in backend

Reply:
move these validations w.r.t account to separate python script `run-account-validations.py` & Extend `test` [github action](https://github.com/rudderlabs/rudder-integrations-config/blob/feat.account_definition/.github/workflows/test.yml#L53-L54) with one more step to trigger `run-account-validations.py`